src/js/Players/ComputerPlayer.js

```diff
@@ -0,0 +1,52 @@
+import {Player} from './Player.js';
+// import {random} from '../../../node_modules/lodash/random.js';
+
+const _ = window._;
+
+export class ComputerPlayer extends Player{
```

Comment:
General comment: there should be a space before "{" at end of line, I suggest to have .editorconfig or eslint to ensure that